doc/common/abbreviations.md

```diff
@@ -15,128 +15,359 @@
+| SATA           | Serial Advanced Technology Attachment |
 | SBA            | Service Based Architecture            |
 | SBI            | South Bound Interface                 |
+| SCAP	         | Security Content Automation Protocol  |
 | SDN            | Software-Defined Networking           |
+| SDNC	         | SDN Controller                        |
+| SDNo	         | SDN Overlay                           |
+| SDNu	         | SDN Underlay                          |
+| SDO	           | Standard Development Organisation     |
 | SDS            | Software-Defined Storage              |
+| SD-WAN	       | Software Defined Wide Area Network    |
+| Sec	           | Security                              |
 | SFC            | Service Function Chaining             |
+| SIEM	         | Security Information and Event Monitoring |
+| SIP            | Session Initiation Protocol           |
 | SLA            | Service Level Agreement               |
-| SMT            | Simultaneous multithreading           |
+| SME            | SUbject Matter Expert                 |
+| SMT            | Simultaneous Multi-Threading          |
+| SNMP	         | Simple Network Management Protocol    |
 | SR-IOV         | Single Root Input Output Virtualisation |
-| SRTM           | Static Root of Trust for Measurements        |
+| SRTM           | Static Root of Trust for Measurements |
 | SSD            | Solid State Drive                     |
+| SSH	           | Secure SHell protocol                 |
+| SSL	           | Secure Sockets Layer                  |
+| SUT	           | System Under Test                     |
 | SW             | Software                              |
+| TC             | Test Case                             |
+| TCP            | Transmission Control Protocol         |
+| TEC            | Telco Edge Cloud (GSMA)               |
+| TIP            | Telecom Infra Project                 |
+| TLB            | Translation Lookaside Buffers         |
 | TLS            | Transport Layer Security              |
 | TOSCA          | Topology and Orchestration Specification for Cloud Applications |
 | TPM            | Trusted Platform Module               |
-| UEFI           | Unified Extensible Firmware Interface  |
+| UEFI           | Unified Extensible Firmware Interface |
 | UI             | User Interface                        |
+| uRLLC	         | ultra-Reliable Low-Latency Communications |
+| VA	           | Virtual Application                   |
 | vCPU           | Virtual CPU                           |
 | VF             | Virtual Function                      |
+| VI             | Vendor Implementation                 |
 | VIM            | Virtualised Infrastructure Manager    |
 | VLAN           | Virtual LAN                           |
 | VM             | Virtual Machine                       |
+| VMM            | Virtual Machine Monitor (or Manager)  |
 | VNF            | Virtual Network Function              |
```

Comment:
We should follow NFV for the NFV terms. 
VNF - Virtualised Network Function, same for VNFC, VNFM

Reply:
According to ETSI GS NFV 003, it is Virtualised Network Function, not Virtual Network Function, etc.

Reply:
@ulikleber Updated as per ETSI GS NFV 003 V1.4.1 (2018-08). Thanks

Reply:
@ulikleber If you agree -- please resolve. Thanks

Reply:
It is done, but didn't show on my screen yet.